src/features/pageObjects/utils/templateFileContent.ts

```diff
@@ -0,0 +1,40 @@
+import { findSubstring } from "../../../common/utils/helpers";
+import { ElementLibrary } from "../../locators/types/generationClasses.types";
+import { PageObject } from "../types/pageObjectSlice.types";
+
+export const editPomContent = (content: string, po: PageObject): string => {
+  let newContent = content;
+  if (po.library === ElementLibrary.MUI && content.indexOf(`<!-- If you need Material UI library -->`) === -1) {
```

Comment:
Should we use newContent here instead of content?

Reply:
I don't see any props and cons for both variants, I'd rather leave it as is